DigitalLearningSolutions.Web/Views/Support/RequestSupportTicket/RequestSummary.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@using DigitalLearningSolutions.Data.Models.Support
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support.RequestSupportTicket;
+@model RequestSummaryViewModel;
+@{
+    bool errorHasOccurred = false;
+    @if (Model.RequestSubject == null && !ViewData.ModelState.IsValid)
+    {
+        errorHasOccurred = true;
+    }
+    ViewData["Title"] = "Type of Request";
+}
+<link rel="stylesheet" href="@Url.Content("~/css/support/support.css")" asp-append-version="true">
+@if (errorHasOccurred)
+{
+    <vc:error-summary order-of-property-names="@(new[] { nameof(Model.RequestSubject),nameof(Model.RequestDescription) })" />
+}
+<div class=" nhsuk-u-margin-top-1">
+    <a class="nhsuk-button nhsuk-button--secondary" role="button" asp-controller="RequestSupportTicket" asp-action="TypeofRequest">
```

Comment:
These Back links should use the [NHS Design System back-link component](https://service-manual.nhs.uk/design-system/components/back-link), as per wireframes.

Reply:
This has been implemented and changes are pushed. Please verify.